### DISCUSSION TITLE
Mahjong Cafe Doll (cafedoll)无法运行的一点资料

### DISCUSSION THREAD

**Discussion by Gently28** [mamedev/discussions#170] [Ideas]
我天真地以为只要rom没有bad,就可能是某个地方没拿完全模拟，我发现cafedoll使用的cafetime的DIP-SWITCHES，而实际它们很多地方不一样，希望以下内容能让可以模拟它的人提供一点帮助。
<img width="4076" alt="cafetime" src="https://github.com/user-attachments/assets/078c4aa9-5817-4daf-9527-c0fb649aa7aa" />

<img width="4464" alt="cafedoll" src="https://github.com/user-attachments/assets/01e78d10-0c58-4f8e-a856-2713aaa15235" />

以下图片不很清晰，但依稀可辨
<img width="602" alt="cafetime (2)" src="https://github.com/user-attachments/assets/43a02c73-c3b2-451d-8225-db13259fd87e" />
![cafedoll (1)](https://github.com/user-attachments/assets/ca2a432a-6d5d-4f3d-9522-6454acbe445c)
![cafedoll (2)](https://github.com/user-attachments/assets/fb67c7ce-3cf7-41f7-b30d-5d96cb0cac2a)
![cafedoll (3)](https://github.com/user-attachments/assets/d6d4f5af-d963-4bfc-ae12-c3534bdef6b7)

**Comment by cuavas** [comment]
Addressed in mamedev/mame@882fad66bdcec6f90a1a0471b896537c5d78cd82
